libs/ui/inputs/src/lib/dropdown-selector/dropdown-selector.component.ts

```diff
@@ -67,11 +69,14 @@ export class DropdownSelectorComponent implements OnInit {
   }
 
   getChoiceLabel(): string {
-    return this.selectedChoice.label
+    return (
+      this.selectedChoice.label.substring(0, 50) +
+      (this.selectedChoice.label.length > 50 ? '...' : '')
+    )
```

Comment:
Can't we achieve this with a `truncate` via CSS?

Reply:
I haven't followed the discussion but TW provide a `truncate` class for ellipsis. It's not about flex.
https://tailwindcss.com/docs/text-overflow#truncate

Reply:
> I haven't followed the discussion but TW provide a `truncate` class for ellipsis. It's not about flex. https://tailwindcss.com/docs/text-overflow#truncate

Yeah I know and use it already, but the flex class on a parent breaks his behaviour.

Reply:
Hum, indeed it doesn't seem simple to make the cascaded `truncate` work within the `flex` here. A `<div class="truncate">` wrapper directly under the flex helps to apply the `truncate` further down, but also cuts borders etc. because of the `overflow-hidden`. 
We really need to keep the `flex justify-end` layout in the `map-view.component`, though, to keep a fitting design for short/normal dropdown entries.

Reply:
@jahow @fgravin What do you think of it ?